deployments/production/spark-2-1-0-master-deployment.json

```diff
@@ -0,0 +1,68 @@
+{
+    "apiVersion": "extensions/v1beta1",
+    "kind": "Deployment",
+    "metadata": {
+        "labels": {
+            "environment": "production",
+            "name": "spark-master-2-1-0",
+            "tag": "2.1.0"
+        },
+        "name": "spark-master-2-1-0",
+        "namespace": "spark-cluster-production"
+    },
+    "spec": {
+        "replicas": 1,
+        "template": {
+            "metadata": {
+                "labels": {
+                    "environment": "production",
+                    "name": "spark-master-2-1-0",
+                    "tag": "2.1.0"
+                }
+            },
+            "spec": {
+                "containers": [
+                    {
+                        "args": [
+                            "/bin/bash",
+                            "/usr/local/spark/sbin/start-master.sh"
+                        ],
+                        "env": [
+                            {
+                                "name": "SPARK_MASTER_HOST",
+                                "value": "spark-cluster-master-2-1-0"
```

Comment:
similar vein as other comments - not sure if we'll be able to listen here if dns doesn't resolve?